src/integration_tests/mission_takeoff_land.cpp

```diff
@@ -0,0 +1,173 @@
+#include <iostream>
+#include <functional>
+#include <memory>
+#include <future>
+#include <atomic>
+#include <cmath>
+#include "integration_test_helper.h"
+#include "path_checker.h"
+#include "mavsdk.h"
+#include "plugins/telemetry/telemetry.h"
+#include "plugins/action/action.h"
+#include "plugins/mission/mission.h"
+
+using namespace mavsdk;
+using namespace std::placeholders; // for `_1`
+
+void do_mission_with_takeoff_and_land(float mission_altitude_m);
+
+TEST_F(SitlTest, MissionTakeoffAndLandHigh)
```

Comment:
Why `High`? What about just:
```suggestion
TEST_F(SitlTest, MissionTakeoffAndLand)
```